assignment/scripts/aboutMe.js

```diff
@@ -46,14 +72,34 @@
 // assigns the highest value to `mostPets`. There's several possibilities --
 // be sure to think through all the scenarios. 
 // console.log `mostPets` after the conditional has run.
+let mostPets;
+if (pets >= friendsPets) {
+    mostPets = pets;
+} else {
+    mostPets = friendsPets;
+}
+console.log('mostPets:', mostPets)
 
 // 17 - Write a *switch* statement that logs:
 //      "First is the worst" if your lucky number is 1
 //      "Second is the best" if your lucky number is 2
 //      "Third is the one with the polka dot dress" if your lucky number is 3
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
+switch (luckyNumber) {
+    case 1:
+        console.log("One is the loneliest number");
+        break; // Break statements needed because JavaScript is weird
+    case 2:
+        console.log("Two can be as bad as one. ",
+                    "It's the loneliest number since the number one.");
```

Comment:
Very LISPy indentation. In JavaScript, you would more typically break at the opening parentheses.

```js
console.log(
  "Two can be as bad as one. ",
  "It's the loneliest number since the number one."
);
```

Reply:
Also, since `console.log` automatically adds a space between arguments, you do not the space after `one.`.